handlers/auth.go

```diff
@@ -18,24 +18,38 @@ import (
 
 // location of the files used for signing and verification
 const (
-	privKeyPath = "keys/teresa.rsa"     // openssl genrsa -out teresa.rsa keysize
-	pubKeyPath  = "keys/teresa.rsa.pub" // openssl rsa -in teresa.rsa -pubout > teresa.rsa.pub
+	privKeyName = "teresa.rsa"     // openssl genrsa -out teresa.rsa keysize
+	pubKeyName  = "teresa.rsa.pub" // openssl rsa -in teresa.rsa -pubout > teresa.rsa.pub
+	secretName  = "teresa-keys"
+	namespace   = "kube-paas"
 )
 
 var (
 	verifyKey *rsa.PublicKey
 	signKey   *rsa.PrivateKey
 )
 
+func getKeyBytes(keyName string) ([]byte, error) {
+	s, err := k8s.Client.Secrets().Get(namespace, secretName)
+	if err != nil {
+		return nil, err
+	}
+	data, ok := s.Data[keyName]
+	if !ok {
+		return nil, fmt.Errorf("Missing key %s", keyName)
```

Comment:
Maybe create a error variable is more idiomatic, like `var ErrMissingKey = errors.New(fmt.Sprintf("Missing key %s", keyName))`, What do you think?